src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+// Actions
+const CHECK_STATUS = 'bookstore/categories/STATUS';
```

Comment:
- [ ] @SSEKPIUS great work on your reducer, however in this case the action type definition doesn't follow the[ ducks pattern](https://github.com/erikras/ducks-modular-redux), the last name in your case 'STATUS' should be equal to the const in your case 'CHECK_STATUS'